src/views/identity/user/modules/NotificationList.vue

```diff
@@ -0,0 +1,154 @@
+    components: {
+        NotificationChannelCard,
+        PPaneLayout,
+        PI,
+        PDivider,
+        PEmpty,
+    },
+    setup() {
+        const vm = getCurrentInstance() as ComponentRenderProxy;
+        const state = reactive({
+            channelList: computed(() => [
+                {
+                    label: vm.$t('IDENTITY.USER.NOTIFICATION.ADD_MEMBER_GROUP'),
+                    link: {
+                        name: IDENTITY_ROUTE.USER.ADD_NOTIFICATION,
+                        params: { channel: 'member' },
+                    },
+                    isProjectOnlyChannel: true,
+                },
+                {
+                    label: vm.$t('IDENTITY.USER.NOTIFICATION.ADD_SMS'),
+                    link: {
+                        name: IDENTITY_ROUTE.USER.ADD_NOTIFICATION,
+                        params: { channel: 'sms' },
+                    },
+                },
+                {
+                    label: vm.$t('IDENTITY.USER.NOTIFICATION.ADD_VOICE'),
+                    link: {
+                        name: IDENTITY_ROUTE.USER.ADD_NOTIFICATION,
+                        params: { channel: 'voice' },
+                    },
+                },
+                {
+                    label: vm.$t('IDENTITY.USER.NOTIFICATION.ADD_SLACK'),
+                    link: {
+                        name: IDENTITY_ROUTE.USER.ADD_NOTIFICATION,
+                        params: { channel: 'slack' },
+                    },
+                },
+            ]),
+            hideProjectOnlyChannel: true,
+        });
+        const routeState = reactive({
+            routes: computed(() => ([
+                { name: vm.$t('MENU.IDENTITY.IDENTITY'), path: '/identity' },
+                { name: vm.$t('MENU.IDENTITY.USER'), path: '/identity/user/account' },
+                { name: vm.$t('IDENTITY.USER.MAIN.NOTIFICATION') },
+            ])),
+        });
+
+        (async () => {
+            if (vm.$route.name === PROJECT_ROUTE.DETAIL.TAB.NOTIFICATIONS) state.hideProjectOnlyChannel = false;
+        })();
+        return {
+            ...toRefs(state),
+            routeState,
+        };
+    },
+};
+</script>
+
+<style lang="postcss" scoped>
+.noti-channel-wrapper {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    width: 100%;
+    height: 100%;
+    padding: 2rem 1rem 2.5rem;
+}
+.sub-title {
+    font-size: 1.375rem;
+    line-height: 145%;
+}
+.channel-list-wrapper {
+    display: grid;
+    row-gap: 0.5rem;
+    column-gap: 0.5rem;
+    grid-template-columns: repeat(auto-fit, minmax(216px, 1fr));
+    margin-top: 1.125rem;
+    margin-bottom: 1.5rem;
+    gap: 0.5rem;
+}
+.channel-item-wrapper {
+    &.hide {
+        display: none;
+    }
+}
+.channel-item {
+    display: flex;
+    flex-direction: column;
+    place-content: center;
+    text-align: center;
+    height: 8.625rem;
+    min-height: 8.625rem;
+    border: 1px solid rgba(theme('colors.gray.300'), 1);
```

Comment:
opacity가 1이면 그냥 

```suggestion
@apply border border-gray-300;
```

이렇게 하는게 좋을것같아요!

Reply:
네네!!